frontend/src/components/UserTicketList/UserTicketList.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Ticket from '../Ticket/Ticket';
+
+const userTicketTypes = [
```

Comment:
@marcianszlovak I guess this is the list of the tickets of users, so its not types
```suggestion
const userTickets = [
```